src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -823,7 +830,7 @@ public void onListUpdate()
 		}
 	}
 
-	private class ChestMonitorHandler<T extends IAEStack> extends MEMonitorHandler<T>
+	private class ChestMonitorHandler<T extends IAEStack<T>> extends MEMonitorHandler<T>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "MEMonitorHandler"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)